node.go

```diff
@@ -86,12 +90,10 @@ func (conn *connection) New(subscripts ...string) (n *node) {
 		panic("YDB: supply node type with at least one string (typically varname)")
 	}
 	n = new(node)
-	n.conn = *conn
+	n.conn = conn
```

Comment:
Note how we just assign the pointer directly. You were dereferencing it, which created a copy.